### ISSUE TITLE
Variable debugger broke

### ISSUE THREAD

**Issue by arthuro555** [4ian/GDevelop#2804]
## Describe the bug
The debugger cannot display variables anymore, displaying the type as undefined and value as null.

## To Reproduce
1. Go on the scene variables
2. Add a new variable `a` with a non-empty value of your choice.
3. Open the debugger and witness the display issue:
![image](https://user-images.githubusercontent.com/19349038/125540300-b7337b70-79c3-4124-b25b-dd72899b925f.png)
4. Use the following JS code in the debugger to verify that the variable is indeed valid and that the issue does come from the debugger: 
```js
const scene = await ((o)=>new Promise((resolve) => {gdjs.RuntimeScene.prototype.renderAndStep=function(...args){o.apply(this, args);resolve(this);}}))(gdjs.RuntimeScene.prototype.renderAndStep);
const v = scene.getVariables().get("a");
[v.getType(), v.getValue()];
```
![image](https://user-images.githubusercontent.com/19349038/125540428-9bfa4f3c-6324-4bad-a987-437816658b55.png)

## Other details
* beta111, bug not present in beta 110
* desktop app
* Windows 10

**Comment by 4ian** [comment]
Looking at this now. This is because of the `ToJSON` method being used when stringifying the variable container.

**Comment by 4ian** [comment]
I think the proper fix is to remove this `toJSON` (we don't want to interfer with this) and replace usage of it with `JSON.stringify(variable.toJSObject())`?

**Comment by 4ian** [comment]
Fixed in 1b965b65a45b7c94b2a7b28d18c888336935f06e
I'll release a new beta soon to avoid too much people to rely on toJSON :)
